styleguide/derek/_partials/solutions/tabs.ejs

```diff
@@ -1,47 +1,308 @@
+          they sold out pickled man braid swag yuccie.
+        </p>
+        <p>
+          Letterpress banh mi narwhal, everyday carry craft beer
+          man braid you probably haven't heard of them. 90's iPhone
+          bushwick hella distillery, artisan tilde readymade before
+          they sold out pickled man braid swag yuccie.
+        </p>
+      </div>
+      <div class="tab-pane" id="tab-4">
+        <p>
+          Activated charcoal blog polaroid, next level four loko
+          listicle vice swag neutra authentic. Actually green juice
+          vape, +1 mlkshk woke meh before they sold out kombucha.
+          Letterpress banh mi narwhal, everyday carry craft beer
+          man braid you probably haven't heard of them. 90's iPhone
+          bushwick hella distillery, artisan tilde readymade before
+          they sold out pickled man braid swag yuccie.
+        </p>
+        <p>
+          Letterpress banh mi narwhal, everyday carry craft beer
+          man braid you probably haven't heard of them. 90's iPhone
+          bushwick hella distillery, artisan tilde readymade before
+          they sold out pickled man braid swag yuccie.
+        </p>
+        <p>
+          Activated charcoal blog polaroid, next level four loko
+          listicle vice swag neutra authentic. Actually green juice
+          vape, +1 mlkshk woke meh before they sold out kombucha.
+          Letterpress banh mi narwhal, everyday carry craft beer
+          man braid you probably haven't heard of them. 90's iPhone
+          bushwick hella distillery, artisan tilde readymade before
+          they sold out pickled man braid swag yuccie.
+        </p>
+        <p>
+          Letterpress banh mi narwhal, everyday carry craft beer
+          man braid you probably haven't heard of them. 90's iPhone
+          bushwick hella distillery, artisan tilde readymade before
+          they sold out pickled man braid swag yuccie.
+        </p>
+      </div>
+  </div>
+</div>
 
-  <!-- Tab panes -->
-  <div id="myTabContent" class="tab-content">
-    <div role="tabpanel" class="tab-pane active" id="tab-1">
-      <div class="container">
-        Content for tab 1
-      </div>
-    </div>
-    <div role="tabpanel" class="tab-pane" id="tab-2">
-      <div class="container">
-        Content for tab 2
-      </div>
-    </div>
-    <div role="tabpanel" class="tab-pane" id="tab-3">
-      <div class="container">
-        Content for tab 3
-      </div>
-    </div>
-    <div role="tabpanel" class="tab-pane" id="tab-4">
-      <div class="container">
-        Content for tab 4
-      </div>
-    </div>
-    <div role="tabpanel" class="tab-pane" id="tab-5">
-      <div class="container">
-        Content for tab 5
-      </div>
-    </div>
-    <div role="tabpanel" class="tab-pane" id="tab-6">
-      <div class="container">
-        Content for tab 6
-      </div>
-    </div>
+<div class="rsTabs-container">
```

Comment:
Will you place some sort of separator, or maybe a comment explaining that this is showing what it looks like with an excessive amount of tabs? It's a little confusing to see where one ends and the other starts.